#! /bin/sh /usr/share/dpatch/dpatch-run
## 110-bugfix-LOGCXX-280.dpatch by  <matthew@localhost>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fixes build problem on HP-UX

@DPATCH@
diff -urNad log4cxx-0.10.0~/src/changes/changes.xml log4cxx-0.10.0/src/changes/changes.xml
--- log4cxx-0.10.0~/src/changes/changes.xml	2008-03-31 15:34:52.000000000 -0700
+++ log4cxx-0.10.0/src/changes/changes.xml	2008-07-17 06:55:12.000000000 -0700
@@ -227,6 +227,7 @@
 <action issue="LOGCXX-257">ServerSocket::accept hangs on Unix</action>
 <action issue="LOGCXX-258">unable to build from make dist package due to missing doxygen file</action>
 <action issue="LOGCXX-259">Several appenders fail to compile in Visual Studio 2008</action>
+<action issue="LOGCXX-280">tests and sample code unnecessarily compiled during default make target</action>
 </release>
 <release version="0.9.7" date="2004-05-10">
 <action type="fix">Fixed examples source code in the "Short introduction to log4cxx".</action>
diff -urNad log4cxx-0.10.0~/src/examples/cpp/Makefile.am log4cxx-0.10.0/src/examples/cpp/Makefile.am
--- log4cxx-0.10.0~/src/examples/cpp/Makefile.am	2008-03-31 15:34:52.000000000 -0700
+++ log4cxx-0.10.0/src/examples/cpp/Makefile.am	2008-07-17 06:53:23.000000000 -0700
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-noinst_PROGRAMS = trivial delayedloop stream console
+check_PROGRAMS = trivial delayedloop stream console
 
 INCLUDES = -I$(top_srcdir)/src/main/include -I$(top_builddir)/src/main/include
 
diff -urNad log4cxx-0.10.0~/src/test/cpp/Makefile.am log4cxx-0.10.0/src/test/cpp/Makefile.am
--- log4cxx-0.10.0~/src/test/cpp/Makefile.am	2008-03-31 15:33:32.000000000 -0700
+++ log4cxx-0.10.0/src/test/cpp/Makefile.am	2008-07-17 06:53:50.000000000 -0700
@@ -24,7 +24,7 @@
 
 INCLUDES = -I$(top_srcdir)/src/main/include -I$(top_builddir)/src/main/include
 
-noinst_PROGRAMS = testsuite
+check_PROGRAMS = testsuite
 
 customlogger_tests = \
 	customlogger/xlogger.cpp\
